### DISCUSSION TITLE
Record video

### DISCUSSION THREAD

**Discussion by fcoayuso** [dgreif/ring#680] [Q&A] [answered]
Hi guys,

I am just wondering if there is a way we can use this API to records the live stream and mimic the ring subscription using internal storage. Has anyone tried that?

Cheers

**Comment by dgreif** [comment, marked as answer]
This is technically possible, and is partially demonstrated in https://github.com/dgreif/ring/blob/master/examples/record-example.ts.  With that said, I personally believe Ring has priced their subscription model very reasonably compared to other competitors, so I generally advise users to use the real subscription.
